app/code/community/Inviqa/SymfonyContainer/Helper/ContainerProvider.php

```diff
@@ -17,12 +17,20 @@ class Inviqa_SymfonyContainer_Helper_ContainerProvider
      */
     private $_generatorConfig;
 
+    /**
+     * @var CompilerPassInterface
+     */
+    private $storeConfigCompilerPass;
 
     public function __construct(array $services = array())
     {
         $this->_generatorConfig = isset($services['generatorConfig']) ?
             $services['generatorConfig'] :
             Mage::getModel('inviqa_symfonyContainer/configurationBuilder')->build();
+
+        $this->storeConfigCompilerPass = isset($services['storeConfigCompilerPass']) ?
```

Comment:
`storeConfigCompilerPass` should be using an underscore, as per Magento conventions:
`$this->_storeConfigCompilerPass`